api/PclusterApiHandler.py

```diff
@@ -695,9 +701,50 @@ def _get_params(_request):
     params.pop("path")
     return params
 
+# Check if parallelcluster tags are active
+def check_tags():
+    try:
+        cost_explorer = boto3.client('ce')
+        inactive_tags = cost_explorer.list_cost_allocation_tags(Status='Inactive',TagKeys=PCLUSTER_COST_TAGS)
+        return {"TagStatus": not (len(inactive_tags['CostAllocationTags']) > 0)}
+    except Exception as e:
+        print("Error %s" % str(e))
+        return {"message": str(e)}, 500
 
-# Proxy
+# Activate parallelcluster tags in cost explorer
+def activate_tags():
+    try:
+        cost_explorer = boto3.client('ce')
+        new_list = [{'TagKey': tag, 'Status': 'Active'} for tag in PCLUSTER_COST_TAGS]
+        updated_tags = cost_explorer.update_cost_allocation_tags_status(CostAllocationTagsStatus=new_list)
+        return updated_tags
+    except Exception as e:
+        return {"message": str(e)}, 500
 
+# Retrieve usage data from Cost Explorer API
+def graph_data():
+    try:
+        cost_explorer = boto3.client('ce')
+        data = cost_explorer.get_cost_and_usage(
+            TimePeriod={
+                'Start': '2022-07-06',
+                'End': '2022-07-12'
```

Comment:
Question: I see these hardcoded values here, is it ok? Since no parameters are taken from the API request, this seems like it always returns values for the time range 2022-07-06 to 2022-07-12, instead of the one selected from the UI

Reply:
This isn't the right commit. Hang on with reviewing, new commit makes this dynamic i.e. today - 7